apps/angular/crud/src/app/global-store.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { Signal } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { EntityId, EntityMap } from '@ngrx/signals/entities';
+import { StateSignal } from '@ngrx/signals/src/state-signal';
+import { of } from 'rxjs';
+import { TodosStore } from './global-store';
+import { Todo, TodoState } from './todo.model';
+import { TodoService } from './todo.service';
+
+describe('TodosStore', () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  let store: {
+    state: Signal<TodoState>;
+    disabled: Signal<boolean>;
+    entityMap: Signal<EntityMap<Todo>>;
+    ids: Signal<EntityId[]>;
+    entities: Signal<Todo[]>;
+    loadAll: () => Promise<void>;
+    updateItem: (todo: Todo) => void;
+    deleteItem: (id: number) => void;
+    resetState: () => void;
+    disableButtons: () => void;
+    enableButtons: () => void;
+  } & StateSignal<{
+    state: TodoState;
+    disabled: boolean;
+    entityMap: EntityMap<Todo>;
+    ids: EntityId[];
+  }>;
```

Comment:
why did you need to write this ?

Reply:
The first part of the assignment is used for linting.
For the StateSignal, It is actually generated by VSCode, which is not necessary for testing.
Thanks for validation. :)